build/build-image/Dockerfile

```diff
@@ -16,6 +16,14 @@
 
 FROM debian:bullseye
 
+ARG USER_ID
+ARG GROUP_ID
+RUN groupadd -g ${GROUP_ID:-1338} agones-dev && \
```

Comment:
On that's very interesting - you set the userid at build time. I always did it at runtime, but this makes FAR more sense.